buerno/Testing- Get data.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Testing- Get data
+  type: http
+  seq: 1
+}
+
+get {
+  url: http://localhost:3000/Ping
```

Comment:
Consider adding error handling for the HTTP request. This can help to debug issues if the request fails for any reason. [important]